### ISSUE TITLE
Prototype Helix ARM64 VM support

### ISSUE THREAD

**Issue by MattGal** [dotnet/arcade#11087]
### Status: waiting on support requests
- [x] Helix - 2100 cores, West US 2.  Support request #: 2209300010002607 (Main environment for testing) (got 500, which is enough to start for sure)
- [x] HelixStaging - 200 cores, West US 2. Support request #: 2209300010003222 (Staging environment for pre-testing before going to main)
- [x] dnceng-internaltooling - 200 cores, West US 2. Support request #: 2209300010001993 (Image generation environment)

### Status: Ubuntu 2004 PR created
- [x] Update dotnet-helix-machines to understand DPSv5 VM skus in various places (CreateCustomImage, DeployQueues, at least)
- [x] Create ubuntu.2004.armarch / ubuntu.2004.armarch.open queues for ARM64 support (we already have on-prem ubuntu.1804.armarch.*; this would keep the existing naming scheme but add lots more capacity and retain the ability to run "whatever" via docker).  : https://dnceng.visualstudio.com/internal/_build/results?buildId=2010535&view=results
- [x] Create windows.11.arm64 / windows.11.arm64.open (the only built-in Azure image currently is Windows 11 preview versions) using the artifacts from the on-prem ARM queues and an Enterprise windows image.  From experimenting we'll likely need a new "Disable OOBE" artifact for the image as a big blocking piece of UI comes up by default.
- [x] Get the above usable in the staging and production environments.

### Open questions:

- Build agents will require a separate 1ES Hosted pool; do we want to generate these?  If so, which flavors of OS?

**Comment by MattGal** [comment]
Created https://dnceng.visualstudio.com/internal/_git/dotnet-helix-machines/pullrequest/26363?_a=files to see how far it gets...

**Comment by MattGal** [comment]
Ubuntu PR merged though I will have to deal with some variance in the UID for docker to work correctly. 
https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/26505 introduces a Windows 11 ARM VM image and may be merged today.

**Comment by MattGal** [comment]
Both images work, but I will need to make a follow-up issue about the problems of docker on ARM64 Azure VMs.  Moved to rollout for now, will link the new issue.

**Comment by MattGal** [comment]
This is complete and rolled out.  Filed https://github.com/dotnet/arcade/issues/11262 to track fixing the UID issue introduced by using Azure VMs
